internal/kafka/test/integration/admin_kafka_suspension_test.go

```diff
@@ -109,18 +110,26 @@ func TestAdminKafka_KafkaSuspension(t *testing.T) {
 	}
 	g.Expect(err).ToNot(gomega.HaveOccurred(), "failed to suspend Kafka instance")
 	g.Expect(privateKafkaReq.Status).To(gomega.Equal(kafkaconstants.KafkaRequestStatusSuspending.String()))
+	err = common.WaitForMetricToBePresent(h, t, metrics.KafkaRequestsStatusCount, kafkaconstants.KafkaRequestStatusSuspending.String())
```

Comment:
can we also check for the metric value? 

The kafka_requests_count metric always produced even if we do not have a Kafka in a given state i.e it'll report `0` if a state doesn't have a Kafka.

What will be good is to check for a particular value: for example
if we change the status to updating the we should expect the value to `1` for that state and `0` for all the other states if we only have one Kafka in the database.
```suggestion
	err = common.WaitForMetricToBePresent(h, t, metrics.KafkaRequestsStatusCount, kafkaconstants.KafkaRequestStatusSuspending.String(), "1")
	err = common.WaitForMetricToBePresent(h, t, metrics.KafkaRequestsStatusCount, kafkaconstants.KafkaRequestStatusSuspended.String(), "0")
	err = common.WaitForMetricToBePresent(h, t, metrics.KafkaRequestsStatusCount, kafkaconstants.KafkaRequestStatusResuming.String(), "0")
```

Same for all the other added checks.